cypress/integration/registration_validation.spec.js

```diff
@@ -188,35 +188,8 @@ describe('User opens registration form and can see validation errors', () => {
     cy.contains(ErrorMessage.MISSING_AUTHOR).should('not.exist');
```

Comment:
(generell kommenter)
har opplevd flere ganger selv også at "The User should be able to see validation errors on contributors tab" blir feil for første bygg, men det funker lokalt og det funker ogås somregel med en retry i aws. vet ikke hvorfor...